### PR TITLE
SG-34066 - Rebranding

### PR DESCRIPTION
**Pull request by juanburgosautoglb**
Change all the shotgrid/shotgun occurrences in the repository.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -2,7 +2,7 @@ SHOTGUN PIPELINE TOOLKIT SOURCE CODE LICENSE
 
 Version: 7/07/2013
 
-Shotgun Software Inc. ("Company") provides the Shotgun Pipeline Toolkit,
+Shotgun Software Inc. ("Company") provides the Flow Production Tracking Toolkit,
```
Comment (carlos-villavicencio-adsk) [inline LICENSE:5]: Does this mean we need to bump the version for this LICENCE (line 3)? @julien-lang

File: python/tk_desktop/ui/about_screen.py [hunk 2]
```diff
@@ -71,10 +71,10 @@ def setupUi(self, AboutScreen):
         QtCore.QMetaObject.connectSlotsByName(AboutScreen)
 
     def retranslateUi(self, AboutScreen):
-        AboutScreen.setWindowTitle(QtGui.QApplication.translate("AboutScreen", "About ShotGrid Desktop", None, QtGui.QApplication.UnicodeUTF8))
-        self.header.setText(QtGui.QApplication.translate("AboutScreen", "<b><big>ShotGrid Desktop</big></b>", None, QtGui.QApplication.UnicodeUTF8))
+        AboutScreen.setWindowTitle(QtGui.QApplication.translate("AboutScreen", "About Flow Production Tracking Toolkit", None, QtGui.QApplication.UnicodeUTF8))
+        self.header.setText(QtGui.QApplication.translate("AboutScreen", "<b><big>Flow Production Tracking Toolkit</big></b>", None, QtGui.QApplication.UnicodeUTF8))
         self.body.setText(QtGui.QApplication.translate("AboutScreen", "Body", None, QtGui.QApplication.UnicodeUTF8))
-        self.copyright.setText(QtGui.QApplication.translate("AboutScreen", "Copyright ©2023 ShotGrid Software Inc.\n"
+        self.copyright.setText(QtGui.QApplication.translate("AboutScreen", "Copyright ©2024 Autodesk.\n"
```
Comment (julien-lang) [inline python/tk_desktop/ui/about_screen.py:77]: No, please don't change this.

**Review by julien-lang — COMMENTED** [review]
Looks pretty good.

**Review by julien-lang — CHANGES_REQUESTED** [review]
Missing asset changes: 

- icon_bg_python.png
- resources folder

File: resources/about_screen.ui [hunk 3]
```diff
@@ -113,7 +113,7 @@
       <string notr="true">font-size: 10px;</string>
      </property>
      <property name="text">
-      <string>Copyright ©2023 ShotGrid Software Inc.
+      <string>Copyright ©2023 Flow Production Tracking Software Inc.
```
[the remark this answers is not in the document]
Comment (julien-lang) [reply resources/about_screen.ui:116]: Let's do Autodesk, Inc. All rights reserved.

File: resources/default_systray_icon.png
Comment (NorberMV) [inline resources/default_systray_icon.png:1]: I believe that icons that are listed in the [resources file](https://github.com/shotgunsoftware/tk-desktop/blob/7293e5d9793283db67b8c746c964cd260fe17b17/resources/resources.qrc#L29C11-L29C35) should be compiled into the `resources_rc.py` file using the [build_resources](https://github.com/shotgunsoftware/tk-desktop/blob/7293e5d9793283db67b8c746c964cd260fe17b17/resources/build_resources.sh#L56) script in order for the changes to be reflected in the SG Desktop UI.

File: resources/wait_screen_icon.png
Comment (NorberMV) [inline resources/wait_screen_icon.png:1]: After taking a quick look at QtDesigner, it seems that the icon is oversized for the QLabel that contains it.
Screenshot 2024-02-05 at 11 13 04 AM

**Review by NorberMV — CHANGES_REQUESTED** [review]

File: resources/build_resources.sh [hunk 4]
```diff
@@ -13,7 +13,7 @@
 # The path to output all built .py files to:
 UI_PYTHON_PATH=../python/tk_desktop/ui
 if [ -z "${PYTHON_BASE}" ]; then
-    # This expects Shotgun Desktop 1.5.9 or less. PySide 1 is required.
+    # This expects Flow Production Tracking Toolkit 1.5.9 or less. PySide 1 is required.
```
Comment (julien-lang) [inline resources/build_resources.sh:16]: I would revert this one because it's historical information.

File: python/tk_desktop/ui/preferences.py [hunk 5]
```diff
@@ -81,13 +81,13 @@ def setupUi(self, Preferences):
 
     def retranslateUi(self, Preferences):
         Preferences.setWindowTitle(QtGui.QApplication.translate("Preferences", "Preferences", None, QtGui.QApplication.UnicodeUTF8))
-        self.auto_start_label.setToolTip(QtGui.QApplication.translate("Preferences", "When checked ShotGrid Desktop will startup automatically when you log in.", None, QtGui.QApplication.UnicodeUTF8))
+        self.auto_start_label.setToolTip(QtGui.QApplication.translate("Preferences", "When checked Flow Production Tracking will startup automatically when you log in.", None, QtGui.QApplication.UnicodeUTF8))
```
Comment (julien-lang) [inline python/tk_desktop/ui/preferences.py:84]:
```suggestion
        self.auto_start_label.setToolTip(QtGui.QApplication.translate("Preferences", "When checked, the PTR desktop app will start automatically when you log in.", None, QtGui.QApplication.UnicodeUTF8))
```

File: python/tk_desktop/ui/preferences.py [hunk 6]
```diff
@@ -81,13 +81,13 @@ def setupUi(self, Preferences):
...
         self.auto_start_label.setText(QtGui.QApplication.translate("Preferences", "Start at login", None, QtGui.QApplication.UnicodeUTF8))
-        self.auto_start_checkbox.setToolTip(QtGui.QApplication.translate("Preferences", "When checked ShotGrid Desktop will startup automatically when you log in.", None, QtGui.QApplication.UnicodeUTF8))
-        self.hotkey.setToolTip(QtGui.QApplication.translate("Preferences", "Enter a hotkey that will auto-raise ShotGrid Desktop when pressed.", None, QtGui.QApplication.UnicodeUTF8))
+        self.auto_start_checkbox.setToolTip(QtGui.QApplication.translate("Preferences", "When checked Flow Production Tracking will startup automatically when you log in.", None, QtGui.QApplication.UnicodeUTF8))
```
Comment (julien-lang) [inline python/tk_desktop/ui/preferences.py:86]:
```suggestion
        self.auto_start_checkbox.setToolTip(QtGui.QApplication.translate("Preferences", "When checked, the PTR desktop app will start automatically when you log in.", None, QtGui.QApplication.UnicodeUTF8))
```

File: python/utils/bootstrap_utilities.py [hunk 7]
```diff
@@ -80,9 +80,9 @@ def emit(self, record):
 
 def _create_proxy(data):
     """
-    Create a proxy based on the data received from the Shotgun Desktop.
+    Create a proxy based on the data received from the Flow Production Tracking.
```
Comment (julien-lang) [inline python/utils/bootstrap_utilities.py:83]:
```suggestion
    Create a proxy based on the data received from the PTR desktop app.
```

File: python/utils/bootstrap_utilities.py [hunk 8]
```diff
@@ -80,9 +80,9 @@ def emit(self, record):
...
 
-    :returns: A connection back to the Shotgun Desktop.
+    :returns: A connection back to the Flow Production Tracking.
```
Comment (julien-lang) [inline python/utils/bootstrap_utilities.py:85]:
```suggestion
    :returns: A connection back to the PTR desktop app.
```

File: resources/banner_widget.ui [hunk 9]
```diff
@@ -33,7 +33,7 @@
 border-color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
-      <string>Welcome to the ShotGrid Desktop. Please &lt;u&gt;click here&lt;/u&gt; to learn more about this app!</string>
+      <string>Welcome to the Flow Production Tracking. Please &lt;u&gt;click here&lt;/u&gt; to learn more about this app!</string>
```
Comment (julien-lang) [inline resources/banner_widget.ui:36]:
```suggestion
      <string>Welcome to Flow Production Tracking. Please &lt;u&gt;click here&lt;/u&gt; to learn more about this app!</string>
```

File: tests/run_command_panel.py [hunk 10]
```diff
@@ -10,7 +10,7 @@
 
 """
 Simple script that allows to test the Project Commands view without having to launch the
-Shotgun Desktop.
+Flow Production Tracking.
```
Comment (julien-lang) [inline tests/run_command_panel.py:13]:
```suggestion
PTR desktop app.
```

File: engine.py [hunk 11]
```diff
@@ -38,7 +38,7 @@ def unhandled_exception_handler(exc_type, exc_value, exc_traceback):
     # Send the message to the logger for this file. In the end, this will
     # get routed to many places:
     # - the log file
-    # - on the site engine side, it will show up in the Shotgun Desktop's console
+    # - on the site engine side, it will show up in the Flow Production Tracking's console
```
Comment (julien-lang) [inline engine.py:41]:
```suggestion
    # - on the site engine side, it will show up in the PTR desktop app's console
```

File: engine.py [hunk 12]
```diff
@@ -221,7 +221,7 @@ def __getattr__(self, name):
                     "Looks like you are trying to run an App that uses a QT based UI, however the "
                     "python installation that the Desktop engine is currently using does not seem "
                     "to contain a valid PySide or PyQt4 install. Either install PySide into your "
-                    "python environment or alternatively switch back to using the native ShotGrid "
+                    "python environment or alternatively switch back to using the native Flow Production Tracking "
                     "Desktop python installation, which includes full QT support."
```
Comment (julien-lang) [inline engine.py:225]: [...]to using the native PTR desktop app python installation[...]

File: python/tk_desktop/licenses.html [hunk 13]
```diff
@@ -1377,9 +1377,9 @@
 <!--   end  Python packages-->
 
 <p><strong>Additional 3rd party Software Credits:</strong><br/>
-    In addition to the 3rd party libraries enumerated above, ShotGrid Desktop integrates various ShotGrid Toolkit
+    In addition to the 3rd party libraries enumerated above, Flow Production Tracking integrates various Flow Production Tracking
     components that are integrating 3rd party libraries. The list of libraries of each component distributed with
-    ShotGrid Desktop can be found here:
+    Flow Production Tracking can be found here:
```
[the remark this answers is not in the document]
Comment (julien-lang) [reply python/tk_desktop/licenses.html:1382]: Missing change here

**Review by julien-lang — CHANGES_REQUESTED** [review]
A few more changes to apply

**Review by julien-lang — APPROVED** [review]
LGTM!

**Review by NorberMV — APPROVED** [review]

**Pull request merged**